stations.csv

```diff
@@ -8948,7 +8948,7 @@ id;name;slug;uic;uic8_sncf;latitude;longitude;parent_station_id;country;time_zon
 10823;Gladbeck West;gladbeck-west;8010019;;51.575357;6.97677;;DE;Europe/Berlin;f;f;f;t;f;f;;;f;;f;8000125;t;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;DEGLA;f;f;;;;;;;;;;;;;글라트베크;;;Гладбек;;;格拉德貝克
 10824;Riedstadt-Goddelau;riedstadt-goddelau;8011246;;49.83323;8.489187;;DE;Europe/Berlin;f;f;f;t;f;f;;;f;;f;8000126;t;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;
 10825;Grävenwiesbach;gravenwiesbach;8011554;;50.382237;8.458552;;DE;Europe/Berlin;f;f;f;t;f;f;;;f;;f;8000132;t;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;Гревенвисбах;;;格雷芬维斯巴赫
-10826;Trier Hbf;trier-hbf;8025181;;49.756848;6.652449;29010;DE;Europe/Berlin;f;t;f;t;f;t;DEZQF;;t;;f;8000134;t;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;DEZQF;f;f;;;;Tréveris ;Trèves;Treviri;;;Trevír;;;トリーア;트리어;Trewir;Tréveris;Трир;;;特里尔
+10826;Trier Hbf;trier-hbf;8025181;;49.756848;6.652449;29010;DE;Europe/Berlin;f;t;f;t;f;t;DEZQF;;t;;f;8000134;t;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;DEZQF;f;f;;;;"Tréveris ";Trèves;Treviri;;;Trevír;;;トリーア;트리어;Trewir;Tréveris;Трир;;;特里尔
```

Comment:
This change looks like it's there by mistake

Reply:
I have opened a PR for this (https://github.com/trainline-eu/stations/pull/462) and will clean this PR.